nuxt.config.js

```diff
@@ -20,7 +20,12 @@ export default {
       { name: 'apple-mobile-web-app-title', content: 'Repair Directory' },
       { name: 'og:site_name', content: 'Repair Directory' },
     ],
-    link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }],
+    link: [
+      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
+
+      // We need to load this font as it is used by REW via the rd_style configuration.
+      { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css?family=Poppins' }]
```

Comment:
I guess in an ideal world we would only load fonts that the current view of the directory is using, but I imagine the cost of implementing that vs any efficiency gain makes it not worth it.

Reply:
And ideally we would also load fonts on demand.  But I couldn't see an easy way to work that out given that this is inserted via nuxt.config, so I took this shortcut.